### ISSUE TITLE
Typo in Pegasus model usage example

### ISSUE THREAD

**Issue by albertovilla** [huggingface/transformers#11942]
## Environment info

- `transformers` version: 4.6.1
- Platform: Linux-5.4.109+-x86_64-with-Ubuntu-18.04-bionic
- Python version: 3.7.10
- PyTorch version (GPU?): 1.8.1+cu101 (True)
- Tensorflow version (GPU?): 2.5.0 (True)
- Using GPU in script?: No
- Using distributed or parallel set-up in script?: No

### Who can help

Documentation: @sgugger

## Information

Model I am using (Bert, XLNet ...): Pegasus

The problem arises when using:
* [x] the official example scripts: (give details below)

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)
* [x] my own task or dataset: (give details below): just trying the Pegasus model

## To reproduce

1. Create a new Colab notebook and install the required libraries using:
```python
!pip install transformers
!pip install sentencepiece
```
2. Copy / paste the "Usage example" from the pegasus [documentation](https://huggingface.co/transformers/model_doc/pegasus.html) page in a cell:

```python
from transformers import PegasusForConditionalGeneration, PegasusTokenizer
import torch

src_text = [
     """ PG&E stated it scheduled the blackouts in response to forecasts for high winds amid dry conditions. The aim is to reduce the risk of wildfires. Nearly 800 thousand customers were scheduled to be affected by the shutoffs which were expected to last through at least midday tomorrow."""
]

model_name = 'google/pegasus-xsum'
device = 'cuda' if torch.cuda.is_available() else 'cpu'

tokenizer = PegasusTokenizer.from_pretrained(model_name)
model = PegasusForConditionalGeneration.from_pretrained(model_name).to(device)

batch = tokenizer(src_text, truncation=True, padding='longest', return_tensors="pt").to(device)
translated = model.generate(**batch)
tgt_text = tokenizer.batch_decode(translated, skip_special_tokens=True)

assert tgt_text[0] == "California's largest electricity provider has turned off power to hundreds of thousands of customers."
```

3. Execute the cell

## Expected behavior

The final statement `assert`  should be True, however there is an typo en the usage example and we get an error:

```python
12 model = PegasusForConditionalGeneration.from_pretrained(model_name).to(device)
     13 
---> 14 batch = tokenizer(src_text, truncation=True, padding='longest', return_tensors="pt").to(torch_device)
     15 translated = model.generate(**batch)
     16 tgt_text = tokenizer.batch_decode(translated, skip_special_tokens=True)

NameError: name 'torch_device' is not defined
```

The reason of the above error is that one of the lines of the code is referring to `torch_device` when it should refer to `device`. Change the line 
- ```python batch = tokenizer(src_text, truncation=True, padding='longest', return_tensors="pt").to(torch_device)```
To the following:
- ```python batch = tokenizer(src_text, truncation=True, padding='longest', return_tensors="pt").to(device)```

**Comment by sgugger** [comment]
Thanks for flagging this! Would you mind sending a PR with the fix since you found it?

**Comment by albertovilla** [comment]
Hi @sgugger, I have just created a PR. 
Kind regards

**Comment by sgugger** [comment]
Thanks!
Closes by #11979
